internal/monitors/collectd/postgresql/postgresql.tmpl

```diff
@@ -0,0 +1,67 @@
+LoadPlugin postgresql
+
+<Plugin postgresql>
+  <Query custom_deadlocks>
```

Comment:
Why is this query specially given here?

Reply:
I don't know the full history, but I think it might have been query that was included in older versions of collectd and later removed.  We added it in our default config.